code/game/objects/structures/flora.dm

```diff
@@ -709,6 +724,7 @@
 /obj/structure/flora/bush/grassy/style_random/Initialize(mapload)
 	. = ..()
 	icon_state = "grassybush_[rand(1, 4)]"
+	update_appearance()
```

Comment:
I hate to see this but making it a component both feels like overkill and also wouldn't make this really any shorter.